osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/messaging/JmsPropertyReader.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.shared.application.config.messaging;
+
+import java.lang.reflect.Method;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.text.WordUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.InvalidPropertyException;
+import org.springframework.core.env.Environment;
+
+public class JmsPropertyReader {
```

Comment:
If this class is used only by other classes from the same package, consider limiting its visibility (and method access) to package-private.